packages/event-handler/src/types/bedrock-agent-function.ts

```diff
@@ -0,0 +1,102 @@
+import type { JSONValue } from '@aws-lambda-powertools/commons/types';
+import type { GenericLogger } from '../types/common.js';
+
+type Configuration = {
+  name: string;
+  definition: string;
+};
+
+// biome-ignore lint/suspicious/noExplicitAny: this is a generic type that is intentionally open
+type Tool<TParams = Record<string, any>> = {
```

Comment:
As mentiioned, technically this function with only take strings because we are not using Zod to parse but in the future we will need the type to be open to accepting any arguments